src/Microsoft.AspNetCore.Mvc.Formatters.Json/MvcJsonOptionsExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    public static class MvcJsonOptionsExtensions
+    {
+        /// <summary>
+        /// Applies camelCasing to property names and dictionary keys.
+        /// </summary>
+        public static MvcJsonOptions UseCamelCasing(this MvcJsonOptions options)
+        {
+            if (options == null)
+            {
+                throw new ArgumentNullException(nameof(options));
+            }
+            
+            options.SerializerSettings.ContractResolver = new CamelCasePropertyNamesContractResolver();
+            
+            return options;
+        }
+
+        /// <summary>
+        /// Property names and dictionary keys are unchanged.
+        /// </summary>
+        public static MvcJsonOptions UseMemberCasing(this MvcJsonOptions options)
+        {
+            if (options == null)
+            {
+                throw new ArgumentNullException(nameof(options));
+            }
+            
+            options.SerializerSettings.ContractResolver = new DefaultContractResolver
```

Comment:
In both cases, I'd change this to check for `DefaultContractResolver` first, so we'll continue to leverage [the shared contract resolver](https://github.com/aspnet/Mvc/blob/f15457c026e2c2cd7cac6030a96c0e08185b9c82/src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonSerializerSettingsProvider.cs#L16-L21):

```csharp
if (options.SerializerSettings.ContractResolver is DefaultContractResolver resolver)
{
    resolver.NamingStrategy = new DefaultNamingStrategy();
}
else
{
    options.SerializerSettings.ContractResolver = new DefaultContractResolver
    {
        NamingStrategy = new DefaultNamingStrategy()
    };
}
```

Reply:
Good catch! I'll add the changes.